src/lib/viewers/controls/color-picker/ColorPickerPalette.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerPalette.scss';
 
 export type Props = {
-    onColorSelect: (color: string) => void;
+    onColorSelect: (color: AnnotationColor) => void;
 };
 
 export default function ColorPickerPalette({ onColorSelect }: Props): JSX.Element {
-    const colors = ['#0061d5', '#26c281', '#ed3757', '#f5b31b', '#ffd700', '#4826c2'];
+    const colors = [
```

Comment:
Should `colors` be passed in as a prop for ColorPickerPalette? That way it'll make this component a lot more reusable for other devs who have other colors that want to use, not just these 6.